modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/view_audit_panel.jsp

```diff
@@ -0,0 +1,32 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<div class="c-p-3" id="<portlet:namespace />-audit-root" %>
+	<div class="inline-item my-5 p-5 w-100">
+		<span aria-hidden="true" class="loading-animation"></span>
+	</div>
+
+	<react:component
+		module="js/PageAuditApp"
+		props='<%=
```

Comment:
I couldn't let this props empty so I added `validConnection` but I don't use it in frontend side. cc: @darquesdev here you have to add the `data` object or a method to inject all the data. Thanks!